### DISCUSSION TITLE
APK Already containers .so... is this alright.. on android?

### DISCUSSION THREAD

**Discussion by jamesmontemagno** [mono/SkiaSharp#1764] [Q&A]
![image](https://user-images.githubusercontent.com/1676321/128657533-4d64883c-a124-4318-9e73-ee53ce02b65c.png)

This is what I am currently seeing. I ensure that any nugets are all compiled agains the same version of SkiaSharp...

**Comment by mattleibow** [comment]
This may as a result of some changes to nuget or my packages. I moved the native bits a round for net6, and I may have resulted in double include. 

It is not bad, something just added a double reference to to native files.

What version of SkiaSharp are you using here? What version of dotnet?

**Reply by jamesmontemagno** [reply]
this is 2.80.2 for both microcharts and the other library i am using. I could go backwards and use an older version too.

This is just with VS 2019 latest bits.

![image](https://user-images.githubusercontent.com/1676321/128738729-6e52c89a-d23c-4683-b166-50e3323dd72b.png)


Seems like since it is ignoring not a big deal.

**Reply by mattleibow** [reply]
I'll have a look and see what pops out?
